sfepy/discrete/fem/mesh.py

```diff
@@ -536,14 +536,14 @@ def create_conn_graph(self, verbose=True):
         output('assembling mesh graph...', verbose=verbose)
         timer = Timer(start=True)
 
-        conn = [self.get_conn(self.descs[0], tdim=k) for k in self.dims]
-        nnz, prow, icol = create_mesh_graph(shape[0], shape[1],
-                                            1, conn, conn)
-        output('...done in %.2f s' % timer.stop(), verbose=verbose)
-        output('graph nonzeros: %d (%.2e%% fill)' \
-               % (nnz, 100.0 * float(nnz) / nm.prod(shape)), verbose=verbose)
+        graph = 0
+        for tdim in self.dims:
+            conn = self.get_conn(self.descs[0], tdim=tdim)
+            graph += create_dof_graph(conn, conn, shape).tocsr()
 
-        data = nm.ones((nnz,), dtype=bool)
-        graph = sp.csr_matrix((data, icol, prow), shape)
+        output('...done in %.2f s' % timer.stop(), verbose=verbose)
```

Comment:
I would use f-strings if possible.

Reply:
Yeah, I left those lines as they were. Lots of places would need such a facelift.